.github/ISSUE_TEMPLATE/documentation-issue.yml

```
@@ -1,12 +1,11 @@
 name: "📚 Documentation Issue"
 description: Report issues in our documentation
-labels: 
-- Issue-Docs
-- Needs-Triage 
+labels: [Needs-Triage, Issue-Docs]
+
 body:
 - type: textarea
-  attributes: 
-    label: Provide a description of requested docs changes
-    placeholder: Briefly describe which document needs to be corrected and why.
   validations:
     required: true
+  attributes: 
+    label: Provide a description of requested docs changes
+    description: Describe which document needs to be corrected and why.
```

I feel the other has a softer tone.
```suggestion
    description: Briefly describe which document needs to be corrected and why you feel this way. 
```